server/bin/pbench-cull-unpacked-tarballs.py

```diff
@@ -428,8 +425,8 @@ def main(options):
             logger,
             options.dry_run,
         )
-        unpacked_dir_name = os.path.basename(tb_incoming_dir)
-        act_name = os.path.join(controller_name, unpacked_dir_name)
+        unpacked_dir_name = Path(tb_incoming_dir).name
+        act_name = Path(controller_name, unpacked_dir_name)
```

Comment:
Nit: it's a path object, not a name.

Reply:
but controller_name isn't a Path obj due to which I will have to create Path object

Reply:
As I said it's a naming nit: if you call something `foo_name`, the implication is that it is the name of the object `foo` (i.e. a string); but `act_name` is not a name, it's not a string; it's a Path object, so maybe call it `act_path`.